changelog.md

```diff
@@ -4,6 +4,10 @@ Since you are interested in what happens next, in case, you work for a for-profi
 
 ---
 
+## 10.0.0-alpha.22 (2019-7-10)
```

Comment:
Please remove the version, this will be automatically added by our build script and put the changelog entry into the `Bug fixes` section